server.js

```diff
@@ -54,6 +54,7 @@ app.use(function (req, res, next) {
         rootURL,
         (webSocketURL || 'wss://ws.blockchain.info'),
         (apiDomain || 'https://api.blockchain.info'),
+        'https://api.blockchain.info',
```

Comment:
Apparently there is no way to get the data from dev, and staging is down. Will it cause problems to have the same apiDomain when on production? @Sjors